sessionctx/variable/sysvar.go

```diff
@@ -486,6 +486,46 @@ var defaultSysVars = []*SysVar{
 		}
 		return normalizedValue, nil
 	}},
+	{Scope: ScopeGlobal, Name: ValidatePasswordEnable, Value: Off, Type: TypeBool},
+	{Scope: ScopeGlobal, Name: ValidatePasswordPolicy, Value: "MEDIUM", Type: TypeEnum, PossibleValues: []string{"LOW", "MEDIUM", "STRONG"}},
+	{Scope: ScopeGlobal, Name: ValidatePasswordCheckUserName, Value: On, Type: TypeBool},
+	{Scope: ScopeGlobal, Name: ValidatePasswordLength, Value: "8", Type: TypeInt, MinValue: 0, MaxValue: math.MaxInt64,
+		Validation: func(vars *SessionVars, normalizedValue string, originalValue string, scope ScopeFlag) (string, error) {
+			var numberCount, specialCharCount, mixedCaseCount int64
+			if numberCountStr, err := vars.GlobalVarsAccessor.GetGlobalSysVar(ValidatePasswordNumberCount); err != nil {
+				return "", err
+			} else if numberCount, err = strconv.ParseInt(numberCountStr, 10, 64); err != nil {
+				return "", err
+			}
+			if specialCharCountStr, err := vars.GlobalVarsAccessor.GetGlobalSysVar(ValidatePasswordNumberCount); err != nil {
+				return "", err
+			} else if specialCharCount, err = strconv.ParseInt(specialCharCountStr, 10, 64); err != nil {
+				return "", err
+			}
+			if mixedCaseCountStr, err := vars.GlobalVarsAccessor.GetGlobalSysVar(ValidatePasswordNumberCount); err != nil {
+				return "", err
+			} else if mixedCaseCount, err = strconv.ParseInt(mixedCaseCountStr, 10, 64); err != nil {
+				return "", err
+			}
+			if length, err := strconv.ParseInt(normalizedValue, 10, 64); err != nil {
+				return "", err
+			} else if length < numberCount+specialCharCount+2*mixedCaseCount {
+				return "", ErrWrongValueForVar.GenWithStackByArgs(ValidatePasswordLength, normalizedValue)
+			}
+			return normalizedValue, nil
+		},
+	},
+	{Scope: ScopeGlobal, Name: ValidatePasswordMixedCaseCount, Value: "1", Type: TypeInt, MinValue: 0, MaxValue: math.MaxInt64},
+	{Scope: ScopeGlobal, Name: ValidatePasswordNumberCount, Value: "1", Type: TypeInt, MinValue: 0, MaxValue: math.MaxInt64},
+	{Scope: ScopeGlobal, Name: ValidatePasswordSpecialCharCount, Value: "1", Type: TypeInt, MinValue: 0, MaxValue: math.MaxInt64},
+	{Scope: ScopeGlobal, Name: ValidatePasswordDictionaryFile, Value: "", Type: TypeStr, SetGlobal: func(_ context.Context, s *SessionVars, val string) error {
```

Comment:
How to use it? The path is on the server side. It's hard to pass the file to k8s, DBASS.